src/utils/bytes.cairo

```diff
@@ -32,8 +33,77 @@ func felt_to_ascii{range_check_ptr}(n: felt) -> (ascii_len: felt, ascii: felt*)
     let ascii_len = [ap - 1];
     let ascii = cast([fp], felt*);
 
-    let (rev: felt*) = alloc();
-    reverse(rev, ascii_len, ascii);
+    reverse(dst, ascii_len, ascii);
 
-    return (ascii_len, rev);
+    return ascii_len;
+}
+
+func bytes_to_bytes8_little_endian(dst: felt*, bytes_len: felt, bytes: felt*) -> felt {
+    alloc_locals;
+
+    if (bytes_len == 0) {
+        return (0);
+    }
+
+    let (local pow256) = get_label_location(pow256_table);
+
+    tempvar dst_index = 0;
+    tempvar bytes_index = bytes_len - 1;
+    tempvar bytes8 = 0;
+    tempvar bytes8_index = 7;
+
+    body:
```

Comment:
this is cairo 0 dark magic haha ;DDD
from the function definition point of view, the signature is way cleaner

but it's slighty harder to read than last version 👀